content/en/docs/tasks/tools/included/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "Tools Included"
+weight: 10
```

Comment:
I don't think we need a weight here as it's used to help order things in menus and whatnot, where these files will be manually included on a page in the order needed.

Reply:
@nate-double-u ah ok thanks - I'd wondered about that, have removed in https://github.com/kubernetes/website/pull/26682/commits/86787c842f37c99109f747fda5a4e833ae7db87c.